script/critest-systemd.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set -eu -o pipefail
+
+report_dir=$1
+
+mkdir -p $report_dir
+BDIR="$(mktemp -d -p $PWD)"
+
+function cleanup() {
+    pkill containerd || true
+    echo ::group::containerd logs
+    cat "$report_dir/containerd.log"
+    echo ::endgroup::
+    rm -rf ${BDIR}
+}
+trap cleanup EXIT
+
+mkdir -p ${BDIR}/{root,state}
+cat > ${BDIR}/config.toml <<EOF
+version = 2
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+runtime_type = "${TEST_RUNTIME}"
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+SystemdCgroup = true
```

Comment:
Since these 2 lines is the only diff with https://github.com/containerd/containerd/blob/main/script/critest.sh. Can we use an env or a variable and use the same `critest.sh` file instead of creating a new one?

Reply:
update, in critest.sh file to check CGROUP_DRIVER var is equal system to dynamic set `SystemdCgroup = true`.